gk/main.c

```diff
@@ -949,7 +953,8 @@ process_gk_cmd(struct gk_cmd_entry *entry, struct gk_add_policy **policies,
 		break;
 
 	case GK_SYNCH_WITH_LPM:
-		gk_synchronize(entry->u.fib, instance);
+		gk_synchronize(entry->u.synch.fib, instance,
+			entry->u.synch.update_only);
```

Comment:
The last parameter is a bool, but `entry->u.synch.update_only` is an `int`. If `entry->u.synch.update_only` is even, it passes `false`. The solution is to pass `!!entry->u.synch.update_only` instead.